packages/adapter-vercel/index.js

```diff
@@ -22,7 +21,7 @@ module.exports = function () {
 			builder.copy_server_files(server_directory);
 			renameSync(join(server_directory, 'app.js'), join(server_directory, 'app.mjs'));
 
-			copy(join(__dirname, 'files'), lambda_directory);
+			builder.copy(join(__dirname, 'files'), lambda_directory);
```

Comment:
I don't see any method called `copy` on the `Builder`

We really need to introduce type checking everywhere